R/miss-x-summary.R

```diff
@@ -4,6 +4,7 @@
 #'   ordering by the most missings in each variable.
 #'
 #' @param data a data.frame
+#' @param order whether or not to order the result by n_miss
```

Comment:
can you please add that this is a logical (TRUE/FALSE), and that TRUE orders from largest to smallest `n_miss`, and that FALSE orders by order provided by the data.